components/idp.js

```diff
@@ -129,17 +139,6 @@ module.run(function(
   brAgreementService, brAuthnService, config) {
   brAgreementService.registerGroup('bedrock-idp.join');
   brAuthnService.displayOrder = ['authn-did', 'authn-password'];
-  // FIXME: need a mechanism for display order on the tabs
```

Comment:
It isn't applicable anymore because this refactor guts out the settings pane logic we used before.